### PR TITLE
[Feature](Nereids)add check to disable unsupported type for Nereids

### PR DESCRIPTION
**Pull request by sohardforaname**
# Proposed changes

Issue Number: close #xxx

## Problem summary

check output slots of all the node and if unsupported type found, throw an exception. 

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

File: regression-test/suites/nereids_p0/aggregate/aggregate.groovy [hunk 1]
```diff
@@ -141,8 +141,8 @@ suite("aggregate") {
     qt_aggregate """ select variance(c_bigint), variance(distinct c_double) from ${tableName}  """
     qt_aggregate """ select 1 k1, 2 k2, c_bigint k3, sum(c_double) from ${tableName} group by 1, k2, k3 order by k1, k2, k3 """
     qt_aggregate """ select (k1 + k2) * k3 k4 from (select 1 k1, 2 k2, c_bigint k3, sum(c_double) from ${tableName} group by 1, k2, k3) t order by k4 """
-    qt_aggregate32" select topn_weighted(c_string,c_bigint,3) from ${tableName}"
-    qt_aggregate33" select avg_weighted(c_double,c_bigint) from ${tableName};"
+    // qt_aggregate32" select topn_weighted(c_string,c_bigint,3) from ${tableName}"
+    // qt_aggregate33" select avg_weighted(c_double,c_bigint) from ${tableName};"
```
Comment (morrySnow) [inline regression-test/suites/nereids_p0/aggregate/aggregate.groovy:145]: avg_weighted do not return array why remove it?

**Review by morrySnow — APPROVED** [review]

File: regression-test/suites/nereids_p0/aggregate/aggregate.groovy (hunk 1 above)
Comment (sohardforaname) [reply regression-test/suites/nereids_p0/aggregate/aggregate.groovy:145]: by mistake, because topn_weighted return array

**Pull request merged**